src/blueapi/worker/task.py

```diff
@@ -10,9 +10,9 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class Task(BlueapiBaseModel):
+class TaskRequest(BlueapiBaseModel):
     """
-    Task that will run a plan
+    Task that will run a plan, different from the WorkerTask
```

Comment:
Should: I don't mind this addition as long as you explain _how_ it is different from the `WorkerTask`

Reply:
ok